ruby_test.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+class TestHound
+  NAME = "Wormbots"
+  VERSION = '0.0.1'
+  LOGISTICS = [NAME, VERSION]
+
+  def foo
+    puts 'bar'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.